app/views/admin/users/show.html.erb

```diff
@@ -35,8 +41,10 @@
     </dl>
   </div>
 
-<%= render "activity" %>
-<%= render "credits" %>
-<%= render "notes" %>
-<%= render "user_organizations" %>
-<%= render "email_tools" %>
+<% if @user.registered %>
```

Comment:
**nitpick (non-blocking):** In other places you use the predicate method, probably best to keep it consistent:

```suggestion
<% if @user.registered? %>
```

Reply:
Great suggestion, @citizen428! 😄